modules/ROOT/pages/cassandra/cassandra-studio.adoc

```diff
@@ -0,0 +1,112 @@
+= Studio - Mule 4
```

Comment:
I don't think this is a best practice as a topic title since it leaves out Cassandra. In google searches, would this show up?

Reply:
Good point, hope this will get passed along to Meera.